index.js

```diff
@@ -187,7 +185,15 @@ module.exports = function(robot) {
     if (queue.isEmpty()) {
       res.send('Nobodyz! Like this: []');
     } else {
-      res.send('Here\'s who\'s in the queue: ' + _.pluck(queue.get(), 'name').join(', ') + '.');
+      res.send('Here\'s who\'s in the queue:');
+      for (var x = 0; x < queue.get().length; x++) {
+        var item = queue.get()[x];
+        if (item.metadata !== undefined) {
+          res.send(item.name + ' with ' + item.metadata + '.');
```

Comment:
let's simplify this into one res.send:

`res.send(item.name + (item.metadata ? '  ' + item.metadata : '') + '.');`